usaspending_api/accounts/serializers.py

```diff
@@ -115,3 +115,11 @@ class Meta:
                 "kwargs": {"read_only": True}
             }
         }
+
+
+class AgenciesFinancialBalancesSerializer(serializers.Serializer):
+
+    fiscal_year = serializers.IntegerField()
+    budget_authority_amount = serializers.DecimalField(21, 2)
```

Comment:
Yes--we think alike on this matter! However, when I removed those precisions, I got this error:
`TypeError: __init__() missing 2 required positional arguments: 'max_digits' and 'decimal_places`

As much as I agree that we don't need to be this specific given the upstream validations, with this precision, we're at $`99,999,999,999,999,999,999.99`. Hopefully we are safe for some time to come!